common/autoconf/flags.m4

```diff
@@ -41,7 +41,11 @@ AC_DEFUN_ONCE([FLAGS_SETUP_INIT_FLAGS],
   if test "x$OPENJDK_TARGET_OS" = xmacosx; then
     ARFLAGS="-r"
   elif test "x$OPENJDK_TARGET_OS" = xaix; then
-    ARFLAGS="-X64"
+    if test "x$OPENJDK_TARGET_CPU_BITS" = "x32"; then
+      ARFLAGS="-X32"
+    else
+      ARFLAGS="-X64"
+    fi
```

Comment:
Things like this need not be conditional:
```
    ARFLAGS="-X$OPENJDK_TARGET_CPU_BITS"
```
and lines 447-455 can be simplified likewise
```
    LDFLAGS_JDK="${LDFLAGS_JDK} -q$OPENJDK_TARGET_CPU_BITS -brtl -bnolibpath -liconv -bexpall -brwexec_must"
    CFLAGS_JDK="${CFLAGS_JDK} -qchars=signed -q$OPENJDK_TARGET_CPU_BITS -qfullpath -qsaveopt -qstackprotect"
    CXXFLAGS_JDK="${CXXFLAGS_JDK} -qchars=signed -q$OPENJDK_TARGET_CPU_BITS -qfullpath -qsaveopt -qstackprotect"
```

Reply:
Updated.